### DISCUSSION TITLE
Can we use Swoole's WebSocket Server over http2 ?

### DISCUSSION THREAD

**Discussion by fakharak** [swoole/swoole-src#4726] [Q&A]
Can we use Swoole's websocket server over http2 ? such that websocket can take advantage of multiplexing of http2.

What i mean is to create a WebSocket Server / Service by setting runtime configurations as something like below:

    'open_http_protocol' => false,
    'open_http2_protocol' => true, // only http2
    'open_websocket_protocol' => true,

**Comment by ClosetGeek-Git** [comment]
Just out of curiosity did you try this?
